attestation-agent/attestation-agent/src/config/mod.rs

```diff
@@ -50,7 +50,7 @@ impl Default for EventlogConfig {
         Self {
             eventlog_algorithm: HashAlgorithm::Sha384,
             init_pcr: DEFAULT_PCR_INDEX,
-            enable_eventlog: false,
```

Comment:
Now only tdx platform supports eventlog recording thus this defaulting to `true` might cause bugs on other platforms.